src/TestFramework/Coverage/XmlReport/JUnitTestFileDataProvider.php

```diff
@@ -81,6 +81,12 @@ public function getTestFileInfo(string $fullyQualifiedClassName): TestFileTimeDa
             );
         }
 
+        $feature = preg_replace("/^(.*):+.*$/", "$1.feature", $fullyQualifiedClassName);
+        if (!$nodes->length) {
```

Comment:
could you please use `$nodes->length === 0` as on line 76 to be consistent here. This is also what PHPStan requires - "only booleans in conditions"

Reply:
Now fixed ✔️